scripts/src/primazactl/cmd/worker/create/namespace.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import traceback
+import sys
+from primazactl.types import kubernetes_name
+from primazactl.primazaworker.workernamespace import WorkerNamespace
+from .constants import SERVICES, APPLICATIONS
+
+
+def add_create_applications_namespace(parser: argparse.ArgumentParser,
+                                      parents=[]):
+    applications_namespace_parser = parser.add_parser(
+        "applications-namespace",
```

Comment:
I'd propose to use the singular form here. This way the final command would be `primazactl main create application-namespace`.

```suggestion
        "application-namespace",
```

Reply:
Fixed